script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2.txt

```diff
@@ -26,14 +26,14 @@ With a smile on her face, Kohaku-san dodges Akiha's question.
 "...Are you sure neither of you noticed?
 Last night, there was a wild dog barking outside for half an hour. Woof woof woof woof, it was so loud."
 <Page7>
-"Aha, you mean like in WanWan Panic, right?"// ワンワンパニック maybe WoofWoof Panic if we're replacing wan with woof? 
+"Aha, you mean like in Woofwoof Panic, right?"
 I couldn't quite tell by her reaction, whether she believes me or not.
 <Page8>
 "...Well, to put it simply, it's something like that."
 "Hmmm―――I do not remember any of that though.
 Neither do you, right, Kohaku?"
 <Page9>
-"I suppose so... I'm sorry, Shiki-san but I don't think that happened."
+"I suppose so... I'm sorry Shiki-san but I don't think that happened."
```

Comment:
why remove this comma?

Reply:
I was trying to get rid of the awkward pause there. "I'm sorry Shiki-san, but I don't think that happened" might be better actually.